### DISCUSSION TITLE
PPC Build?

### DISCUSSION THREAD

**Discussion by dylang-usask** [quicksilver/Quicksilver#2949] [General]
I'm setting up an old Powerbook G4 I have, and I'm looking for a copy of Quicksilver ß60(released 05 Jun 2011), which will work on it.  Unfortunately, qsapp.com doesn't have legacy downloads going back that far.  I've even tried using the Internet Archive to retrieve the old DMGs, but the DMGs are not a part of the snapshot (see https://web.archive.org/web/20170301180428/https://qsapp.com/archives/   and  https://web.archive.org/web/20170301180428/https://qsapp.com/archives/downloads/Quicksilver%20B60.dmg ).

Is there anyway to get a Quicksilver ß60 DMG from anywhere?  Thanks.

**Comment by dylang-usask** [comment]
https://macintoshgarden.org/apps/quicksilver

I did find this resource, but there is no B60 prebuilt for PPC.

**Comment by skurfer** [comment]
I forgot we moved the archive link to point to GitHub. The original archive page had those old versions I think. I thought I had local copies, but haven’t found them.

I’ll try to check my backups when I have a chance or maybe they’re still on the server where we host qs0. Or maybe @pjrobertson has the old archive folder somewhere?

**Comment by n8henrie** [comment]
I don't see any old versions on the current server.

```console
$ find / -iname '*.dmg*' 2>/dev/null
$
```

It looks like access to the old server is gone.

@dylang-usask -- any chance you can get Xcode installed and could compile from source?

Alternatively, it looks like there is an old version available for download at <https://www.macintoshrepository.org/26385-quicksilver>; the download works, and my locally downloaded hash matches what they provide:

```
$ sha1sum Quicksilver-B60.zip
1f6d4fb853d174cacd35ac76958450dd08f8360e  Quicksilver-B60.zip
```

Unfortunately I don't have any idea what the original hash of this might have been, so it's entirely possible that it has been tampered with, and as such I **don't** condone using it -- caveat emptor!

FWIW here is the relevant tag; I don't know if a hash was included anywhere for comparison: <https://github.com/quicksilver/Quicksilver/releases/tag/B60>

**Comment by n8henrie** [comment]
FWIW, I don't see any other results on Google for the matching md5 or sha256, and that link is the only result for the sha1.

```console
$ md5sum Quicksilver-B60.zip
9660fccaa575cdb7f2bf7e0922de31e1  Quicksilver-B60.zip
$ sha256sum Quicksilver-B60.zip
0b35c4b6684a924020c29f5c2a06bf41aedf341335a1bbb796830f8bb6183a37  Quicksilver-B60.zip
```

**Comment by n8henrie** [comment]
FWIW, hash matches that of <https://archive.org/download/Macintosh_Garden_Apps_Q_2019/Quicksilver-B60.zip>
